client/termdb/tree.js

```diff
@@ -321,30 +335,87 @@ function setRenderers(self) {
 		select(this).style('display', 'none')
 	}
 
+	self.renderSampleTable = function () {
```

Comment:
@airenzp I suggest moving the sample table renderer functions at the end of `self.renderers()`, since these logically happen only after the tree menu is rendered and we want to keep the tree branch renderer function closer together.